### ISSUE TITLE
a problem of sidebar when article catalog contains Chinese

### ISSUE THREAD

**Issue by Waldenth** [liuyib/hexo-theme-stun#102]
## I want to create a new issue 

- [x] Yes, I have read [FAQ](https://github.com/liuyib/hexo-theme-stun/blob/master/FAQ.md).
- [x] Yes, I have read [Hexo Docs page](https://hexo.io/docs/).
- [x] Yes, I have read [Hexo-theme-stun Docs](https://liuyib.github.io/hexo-theme-stun/zh-CN/).
- [x] And yes, I already searched for current [issues](https://github.com/liuyib/hexo-theme-stun/issues) and this did not help me.

## Stun Information

**Stun Version:** 2.1.1 

**Hexo Version:** 5.2.0 

**Platform:**  Windows 10

**Browser:**  Chrome / FireFox

## Expected behaviour 
I write a article and upload to my blog, but the article's each catalog is Chinese, When I browse the article by Chrome Browser, I found the sidebar component just show the primary catalog, but when the catalogs are English , the sidebar component works fine. However , when I look the sample blog in the README, I found the sidebar can work when article has Chinese catalog. It puzzles me. 

**English Catalog (work fine**
![](https://i.loli.net/2020/09/27/oEumKAQbI1fa7h3.png)
**Chine Catalog  (work fine**
![](https://i.loli.net/2020/09/27/f7kXMDGHavgYPyV.png)

## Actual behaviour 

**Chinese Catalog (wrong**
![](https://i.loli.net/2020/09/27/HstAgF3bpVkjJiG.jpg)
My wrong webpage , the sidebar just shows the primary catalog .
## Steps to reproduce the behaviour 
My article is writen by `Notepad++` and `Typora`,so I do not think it is caused by Windows Platform , when I look the webpage's source code, I find something interesting. In normal webpage , the souce code `href` url is Chinese too , but in the wrong webpage , the source code `href` url is escape character. The following are screenphotos.
**wrong page**
![wrong page](https://i.loli.net/2020/09/27/HxU7zmWVQ9c38pX.png)

**normal page**
![](https://i.loli.net/2020/09/27/sIkQpdTnGyZa4Hf.png)
So I think the problem maybe in here. But I do not know how to fix it. Maybe my hexo setting is not right , but I can not find useful information on the Internet. I have also checked my files and all of them are encoding UTF-8 ,and the webpage source code head is
```html
<!DOCTYPE html><html lang="zh-CN"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1">
```
my HEXO package.json is
```javascript
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "5.2.0"
  },
  "dependencies": {
    "hexo": "^5.0.0",
    "hexo-deployer-git": "^2.1.0",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-index": "^2.0.0",
    "hexo-generator-search": "^2.4.1",
    "hexo-generator-tag": "^1.0.0",
    "hexo-render-pug": "^2.1.4",
    "hexo-renderer-ejs": "^1.0.0",
    "hexo-renderer-markdown-it-plus": "^1.0.4",
    "hexo-renderer-pug": "^1.0.0",
    "hexo-renderer-stylus": "^2.0.0",
    "hexo-server": "^2.0.0"
  }
}
```
when I click the  escape character url in the source code, it can correctly jump to the right catalog, and the url in the browser is right too.
**click it**
![](https://i.loli.net/2020/09/27/k1vJaQbLdi74AXE.png)
**jump successfully**
![](https://i.loli.net/2020/09/27/1E2sxC6l7ruqWoN.png)
Maybe it's just my personal problem. And my statement may be a little unclear, because I'm not familiar with Web frameworks. By the way , whether it can be solved or not, thank you very much!

## Feature Request 

---

**Comment by liuyib** [comment]
@Waldenth
Hi Waldenth, I have confirmed that this is a problem with the Stun theme. I will fix it as soon as possible and sincerely thanks to your feedback. ❤️

**Comment by liuyib** [comment]
@Waldenth fixed: see: https://github.com/liuyib/hexo-theme-stun/commit/729d89ea758e1ef55fd0b2b7430856904d19763b

**Comment by Waldenth** [comment]
> @Waldenth fixed, see: [729d89e](https://github.com/liuyib/hexo-theme-stun/commit/729d89ea758e1ef55fd0b2b7430856904d19763b)

Thank you very much , now it works perfectly.

**Comment by Waldenth** [comment]
> @Waldenth fixed, see: [729d89e](https://github.com/liuyib/hexo-theme-stun/commit/729d89ea758e1ef55fd0b2b7430856904d19763b)

@liuyib Thank you very much , now it works perfectly.😁
